nss-day-cohort-25-website/nss-day-cohort-25-website
# Rm update student cards

# Description
Updated student cards with student modal that gets called on clicking the info button for a particular student card.

Please include a summary of the change. List any dependencies that are required for this change.

To Test

```
git fetch --all
git checkout rm-update-student-cards
```
1. pull down code
1. run npm start from root directory
1. confirm that main page still loads correctly
1. click on the info button on a students card
1. a modal should appear with that students information
1. clicking either the transparent black background, or the close button should close the modal
1. click on more student cards to confirm that new students are loaded into the modal correctly

Fixes #2 

## Type of change

Please delete options that are not relevant.
